src/Convert.cpp

```diff
@@ -199,42 +193,61 @@ QFont fontDeclarationToFont(const QString& fontDecl)
 
 struct Undefined {
 };
-using ExprValue = boost::variant<Undefined, QColor, QUrl>;
+using ExprValue = std::variant<Undefined, QColor, QUrl>;
+
+int lexicalCastInt(const std::string& s)
+{
+  if (s.empty() || ((!isdigit(s[0])) && (s[0] != '-') && (s[0] != '+')))
```

Comment:
I don't understand the reason for checking the first character. `p` will point to it, if a conversion failed.
```suggestion
  if (s.empty())
```

Reply:
It was just there as an edge case check, like if the user entered the number "-1 or +1" but perhaps unnecessary in this context.  I can remove the extra checks.

Reply:
I found out that `std::strtol` ignores all leading whitespace characters while `boost::lexical_cast` does not. If this functions is meant behave exactly like `boost::lexical_cast`, then your additional check was correct. I'm sorry.

Reply:
Yes my goal was to replace the boost functionality as exactly as possible so as not to break any of the functionality.  I thought maybe you knew something I didn't that might make the additional checks unnecessary.  If not I will revert back to the original version to be safe.